README.md

```diff
@@ -61,6 +61,21 @@ The page text should be formatted with HTML tags.
 
 ![screenshot of add-page](images/updatepage.png)
 
+### Task: Update confluence environment page
```

Comment:
I could not configure the target page correctly to make this update work.  I tried a 2-row, 3-column table in this fashion:
Environment, Application Name, Version
Demo
But the updates failed.  Can you provide a screenshot of the target page and the raw html for it?